arches/app/models/migrations/10516_rename_geometrynodeid_spatialview_geometrynode_et_al.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 4.2.10 on 2024-03-22 19:24
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('models', '10515_i18n_plugins'),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name='spatialview',
+            old_name='geometrynodeid',
+            new_name='geometrynode',
+        ),
```

Comment:
@whatisgalen can we keep the db_column name the same (`geometrynodeid`)and just change the django model name? I think this will reflect things better when working on the db functions. Also means I don't have to refactor a load of the db functions!!

The django model change will be fine.

`geometrynode = models.ForeignKey(Node, on_delete=models.CASCADE, db_column="geometrynodeid")`